cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/MeasureScoringTypeCohortTest.java

```diff
@@ -89,22 +87,14 @@ void cohortResourcePopulation() {
 
     @Test
     void cohortBooleanMissingRequiredPopulation() {
-        try {
-            given.when()
-                    .measureId("CohortBooleanMissingReqdPopulation")
-                    .evaluate()
-                    .then()
-                    .firstGroup()
-                    .population("initial-population")
-                    .hasCount(10)
-                    .up()
-                    .up()
-                    .report();
-            fail("This should throw error");
-        } catch (UnsupportedOperationException e) {
-            assertTrue(e.getMessage()
-                    .contains("MeasurePopulationType: numerator, is not a member of allowed 'cohort' populations"));
-        }
+        given.when()
+                .measureId("CohortBooleanMissingReqdPopulation")
+                .evaluate()
+                .then()
```

Comment:
Question:  are you expecting this operation to have a status of ERROR like the other tests?